samples/Terraform Template/os_alpine_3_17_userdata.tftpl

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env sh
```

Comment:
This is going to cause troubles as this is a shell interpreter not a Terraform interpreter and we can't have two languages sharing an interpreter. It's probably best to remove this sample entirely.

Reply:
@lildude Oh yes you're right.  I'll look at & fix tomorrow.  Thanks.